packages/react-core/src/Authenticator/hooks/useAuthenticator/__tests__/useAuthenticator.spec.tsx

```diff
@@ -1,15 +1,53 @@
 import React from 'react';
 import { renderHook } from '@testing-library/react-hooks';
-import * as utils from '../utils';
+import { AuthenticatorServiceFacade } from '@aws-amplify/ui';
+import * as UIModule from '@aws-amplify/ui';
 
 import { AuthenticatorProvider } from '../../../context';
 import { USE_AUTHENTICATOR_ERROR } from '../constants';
+import * as utils from '../utils';
 import { useAuthenticator, UseAuthenticator } from '..';
 
+const mockServiceFacade: AuthenticatorServiceFacade = {
+  authStatus: 'authenticated',
+  codeDeliveryDetails: {} as UseAuthenticator['codeDeliveryDetails'],
+  error: undefined as unknown as UseAuthenticator['error'],
```

Comment:
Just curious if there's a cleaner way to cast this

Reply:
Not really - `AuthenticatorServiceFacade` requires `error` field to be of type `string` so the cast to `unknown` is required unless we update the `AuthenticatorServiceFacade` type